packages/dev-utils/test/blockchain_lifecycle_test.ts

```diff
@@ -8,8 +8,8 @@ import { BlockchainLifecycle, web3Factory } from '../src';
 const expect = chai.expect;
 
 describe('BlockchainLifecycle tests', () => {
-    const web3Provider = web3Factory.getRpcProvider();
-    const web3Wrapper = new Web3Wrapper(web3Provider);
+    const provider = web3Factory.getRpcProvider({ shouldUseInProcessGanache: true });
```

Comment:
Seems like we have `shouldUseInProcessGanache: true` spread across 4 different packages, is `dev-utils` the rightful place for this to live? Looks like all other projects use the web3Factory from dev-utils, might as well have one place for the Ganache-ness to live.

Reply:
It's only in one place per package. Think that's fine.